examples/ProtoBuffer/protobuffer.ino

```diff
@@ -0,0 +1,70 @@
+#include <TheThingsNetwork.h>
+
+const byte appEui[8] = { 0x70, 0xB3, 0xD5, 0x7E, 0xD0, 0x00, 0x11, 0x86 };
+const byte appKey[16] = { 0xEF, 0x4C, 0xAA, 0x85, 0xE1, 0xB2, 0x26, 0x53, 0x98, 0xE2, 0x3C, 0xD1, 0x31, 0x50, 0xCA, 0xDF };
+
+#define loraSerial Serial1
+#define debugSerial Serial
+
+TheThingsNetwork ttn(loraSerial, debugSerial, TTN_FP_EU868);
+TheThingsMessage msg(debugSerial);
+
+api_Measurement measurement = api_Measurement_init_default;
+
+void setup()
+{
+  pinMode(2, INPUT);
+
+  loraSerial.begin(57600);
+  debugSerial.begin(9600);
+
+  // Wait a maximum of 10s for Serial Monitor
+  while (!debugSerial && millis() < 10000);
+
+  //Set callback for incoming message
+  msg.onMessage(message);
```

Comment:
Leave it `ttn.onMessage()` and in the the callback let user do `TheThingsMessage msg(payload, size, port)`